tinyagentos/projects/task_store.py

```
@@ -198,6 +199,15 @@ async def _post_init(self) -> None:
             "ON project_tasks(project_id, element_id)"
         )
         await self._db.commit()
+        # Add created_by column for checklist items (defect tsk-6xymzj)
+        try:
+            await self._db.execute(
+                "ALTER TABLE task_checklist_items ADD COLUMN created_by TEXT"
+            )
+            await self._db.commit()
+        except Exception:
+            # Column already exists on fresh installs (created by SCHEMA).
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)